docs/index.md

```diff
@@ -0,0 +1,109 @@
+# Pay with Card
+
+Accept card payments in your app using the PayPal In-App Payments SDK.
+
+## How it works
+
+The Payments SDK allows for merchant apps to integrate easily with PayPal payment services on both the client and server.
+
+You will need a server integration to create an order and capture the funds using [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). The Payments SDK allows for client-side approval of an order with the user's card information.
+
+## Eligibility
+```
+# Placeholder: Product Eligibility
+what kind of merchants are we targetting?
+what features do we offer right now/what features are we missing?
+```
+
+## Integration methods
+
+- [Card Fields UI Integration](#card-fields-ui-integration)
+- [Fully Customizable Integration](#fully-customizable-integration)
+
+## Card Fields UI Integration
+
+Accept card payments using our card fields UI
+
+```
+# Placeholder: Include images of card fields
+```
+
+**Requirements:**
+Card fields UI offers a low effort integration where you can use our card fields UI, and our card fields will handle ...
+
+## Fully Customizable Integration
+
+Process card payments with PayPal Payments SDK using your own UI.
+
+### Steps
+
+#### 1. Create an order
+Call `v2/checkout/orders` to create an order an obtain an order ID
+
+```
+curl --location --request POST 'https://api.sandbox.paypal.com/v2/checkout/orders/' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer <access_token>' \
+--data-raw '{
+    "intent": "CAPTURE",
+    "purchase_units": [
+        {
+            "amount": {
+                "currency_code": "USD",
+                "value": "5.00"
+            }
+        }
+    ]
+}'
+```
+
+#### 2. Approve the order using Payments SDK:
+
+```swift
+import PaymentsSDK
+
+// Create a config object using your clientID obtained from PayPal application portal
+let config = CoreConfig(clientID: "<CLIENT_ID>", evironment: .sandbox)
+
+// Create a CardClient to approve an order with a card payment method
+let cardClient = CardClient(config: config)
+
+// Create a card object
+let card = Card(
```

Comment:
this probably is not literally how someone would code it as they will get the inputs from some UI - would it make sense to separate out the steps such that it is

1. Show your Card Fields  
2. Render "Pay now" button
3. On submit >  Create + Approve Order
4. On success > Authorize or Capture depending on intent during Order creation

Is there any example of anyone else articulating these steps

Reply:
We are making several assumptions here since each parameter is of type `String`, we assume that the developer knows where to access the string content necessary to construct a `Card` object.

Reply:
I agree some visual elements may be helpful, we could provide an image of a mock UI alongside the code samples to streamline the code snippets.

Reply:
@sarahkoop and I talked about it and we feel that we should move this forward because we are not offering client-side order creation